massa-metrics/src/lib.rs

```diff
@@ -17,7 +17,7 @@ use prometheus::{register_int_gauge, Gauge, Histogram, IntCounter, IntGauge};
 use tokio::sync::oneshot::Sender;
 use tracing::warn;
 
-// #[cfg(not(feature = "testing"))]
+// #[cfg(not(feature = "test-exports"))]
```

Comment:
It's commented out, maybe remove instead

Reply:
Fixed.